packages/@aws-cdk/aws-route53-patterns/lib/website-redirect.ts

```diff
@@ -11,23 +11,34 @@ import * as crypto from 'crypto';
  */
 export interface HttpsRedirectProps {
   /**
-   * HostedZone of the domain
+   * Hosted zone of the domain. Container for records which contain information
+   * about how you want to route traffic for a specific domain (example.com)
+   * and it's subdomains (acme.example.com, zenith.example.com). 
+   * 
+   * A hosted zone and the corresponding domain will have the same name.
    */
   readonly zone: IHostedZone;
+
   /**
-   * The redirect target domain
+   * The redirect target fully qualified domain name (FQDN). An alias record
+   * will be created that points to your CloudFront distribution. Root domain
+   * or sub-domain can be supplied.
    */
   readonly targetDomain: string;
+
   /**
-   * The domain names to create that will redirect to `targetDomain`
+   * The domain names that will redirect to `targetDomain`
    *
-   * @default - the domain name of the zone
+   * @default - the domain name of the hosted zone
    */
   readonly recordNames?: string[];
+
   /**
-   * The ACM certificate; Has to be in us-east-1
+   * The AWS Certificate Manager (ACM) certificate that will be associated with
+   * the CloudFront distribution that will be created. The certificate must be
+   * stored in us-east-1 (N. Virginia)
    *
-   * @default - create a new certificate in us-east-1
+   * @default - A new X509 certificate is created using AWS Certificate manager in us-east-1 (N. Virginia)
```

Comment:
```suggestion
   * @default - A new certificate is created in us-east-1 (N. Virginia)
```

Also, I just noticed you're not actually enforcing this at runtime. I just did this for the new `Distribution` construct for CloudFront [here](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-cloudfront/lib/distribution.ts#L137-L142).

```ts
    if (props.certificate) {
      const certificateRegion = Stack.of(this).parseArn(props.certificate.certificateArn).region;
      if (!Token.isUnresolved(certificateRegion) && certificateRegion !== 'us-east-1') {
        throw new Error('Distribution certificates must be in the us-east-1 region and the certificate you provided is in $Region.');
      }
    }
```

This will be redundant when this class switches over from `CloudFrontWebDistribution` to `Distribution`, but in the meantime might save some headache (and defense in depth is always nice).